aries-site/src/pages/components/search.mdx

```diff
@@ -64,15 +64,26 @@ Placeholder text should only contain the text "Search".
 
 ### Accessibility
 
-Search inputs should be accessible. Ensure users have the ability to return results using their keyboard as well as clicking a button.
+Search inputs should be accessible. Ensure users have the ability to return results using their keyboard or by clicking a button.
 
-Inputs should always be accompanied by labels for accessibility. An icon may serve as a label if:
+All search inputs should specify `type="search"`.
 
-1. The icon meaning is well understood.
-2. Has an 'aria-labelledby' attribute.
+If the search input is within a form then the placeholder should be used instead of a label. If the search input is outside of a form use an`aria-label` in addition to a placeholder.
```

Comment:
Can you link to resource of where it's appropriate for placeholder text to serve as replacement for label? This feels off from my understanding. If anything, it feels like we should always suggest an aria-label unless a visual label in a form is provided.

Reply:
Link to a Figma conversation about this https://www.figma.com/file/KamqhSB2V8TutRsGMAuGgi?node-id=3:1011&mode=design#683671697. 

I think my message might have accidentally been misread, a placeholder can **not** be used instead of a label.

Reply:
Thanks for picking up on that Taylor. I must have mis read the comment.